package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "axios": "^1.0.0",
+    "bull": "^4.10.4",
```

Comment:
would like to know which other alternatives were considered and what criteria were used to finalize this package. If there are any documents or findings which led you to select this package, would be great if you can share them as well.

Reply:
https://npmcompare.com/compare/agenda,bee-queue,bull,kue,later 
https://blog.logrocket.com/comparing-best-node-js-schedulers/
Some of the resource I went through to find best suitable one for the requirements